examples/plot_loco_minimal.py

```diff
@@ -0,0 +1,143 @@
+"""
+Leave-One-Covariate-Out (LOCO) feature importance with different regression models
+==================================================================================
+
+This example demonstrates how to compare LOCO feature importance :footcite:t:`Williamson_General_2023` across different
+predictive models on the same regression dataset. LOCO is model-agnostic and can be
+applied to any predictive model. Here, we use a linear model, a random forest, a neural
+network, and a support vector machine. We compare the models based on their predictive
+performance (R2 score) and the LOCO feature importance they yield.
+"""
+
+# %%
+import pandas as pd
```

Comment:
I think it is not great to burden the reader with 8 lines of import. 
Let's do the import where they are needed.

Reply:
Yes, it's clearer this way.